ansible/deploy.yml

```diff
@@ -0,0 +1,48 @@
+---
+- hosts: hypervisors
+  tasks:
+    - include_tasks: host_setup.yml
+
+- hosts: controllers
```

Comment:
Shouldn't we do this on the hypervisors? Or localhost? Possibly also using `run_once`.

Reply:
I guess `controllers==localhost` explains this comment too.